src/app/views/staff/staff.controller.coffee

```diff
@@ -15,16 +16,29 @@
   MnoeUsers.get($stateParams.staffId).then(
     (response) ->
       vm.staff = response.data
+      if vm.staff.sub_tenant_id
+        MnoeSubTenants.get(vm.staff.sub_tenant_id).then(
+          (result) ->
+            vm.staff.subTenantName = result.data.name
```

Comment:
Should we store the `SubTenant` response as an attribute of the view so it is easily exposed?
```
vm.sub_tenant = result.data
```

Reply:
Corrected